ipfx/data_set_features.py

```diff
@@ -124,7 +126,9 @@ def extract_sweep_features(data_set, sweep_table):
         sweep_numbers = sorted(sweep_numbers)
         logging.debug("%s:%s" % (stimulus_name, ','.join(map(str, sweep_numbers))))
 
-        sweep_set = data_set.stim_aligned_sweep_set(sweep_numbers)
+        sweep_set = data_set.sweep_set(sweep_numbers)
+        sweep_set.select_epoch("response")
+        stim_align_sweep_set(sweep_set)
```

Comment:
It looks like this pattern of:

```
x_set = data_set.sweep_set()
x_set.select_epoch('response')
stim_align_sweep_set(x_set)
```
is repeated for all the changes in this commit. Consider creating a function to avoid repetition.